src/models/models.ts

```diff
@@ -26,18 +26,95 @@ export enum AllianceColor {
 
 export enum Topic {
 	auto = 'Auto',
-	collection = 'Collection',
-	shooting = 'Shooting',
-	amp = 'Amp',
-	path = 'Path',
-	defense = 'Defense',
-	endgame = 'Endgame',
-	humanPlayer = 'Human Player',
-	penalties = 'Penalties',
+	pathing = 'Pathing',
+	coral = 'Coral',
+	algae = 'Algae',
 	drivers = 'Drivers',
+	hp = 'Human Player',
+	penalties = 'Penalties',
+	climb = 'Climb',
+	defense = 'Defense',
 	other = 'Other'
 }
 
+export enum Auto {
```

Comment:
Let's be more specific with these enum names, like "AlgaeGroundCollection" instead of "Algae1", or "AutoAccuracy" instead of just "Auto". The increased specificity will help code readability, so it'll be easier to make changes in the future/next year.